website/source/api/secret/pki/index.html.md

```diff
@@ -318,47 +318,47 @@ $ curl \
 
 ## Set URLs
 
-This endpoint allows setting the issuing certificate endpoints, CRL distribution
-points, and OCSP server endpoints that will be encoded into issued certificates.
-You can update any of the values at any time without affecting the other
-existing values. To remove the values, simply use a blank string as the
-parameter.
+<!--apidoc_start path:/pki/config/urls method:post -->
+<!-- This section is auto-generated and must be updated in the Go source file. -->
 
-| Method   | Path                         |
-| :--------------------------- | :--------------------- |
-| `POST`   | `/pki/config/urls`           |
-
-### Parameters
+This endpoint allows setting the issuing certificate endpoints, CRL distribution 				points, and
+OCSP server endpoints that will be encoded into issued certificates. 				You can update any of the
+values at any time without affecting the other 				existing values. To remove the values, simply use
+a blank string as the 				parameter.
```

Comment:
After I ran `make apidoc` on my Ubuntu machine, it edited this file to look like this. Is that expected for the tool? Will it still render just fine or did I do something wrong by re-running it on something that had already been formatted?

Reply:
Hmmm, I don't get that. When I ran it just now it undid that change. Can you try again (I just pushed the correctly formatted version)?